### ISSUE TITLE
Experiment: Vertex AI Training Run - 10K Episodes CPU

### ISSUE THREAD

**Issue by LiamConnell** [LiamConnell/FootyAI#9]
## Experiment Configuration
- **Job ID**: `825863610487013376`
- **Start Time**: 2025-07-24 20:11 UTC
- **Total Episodes**: 10,000 
- **Current Progress**: ~1,200/10,000 episodes (12%)
- **Machine Type**: n1-standard-4 (CPU only)
- **Batch Size**: 32 environments
- **Learning Rate**: 1e-3
- **Git Commit**: `2792431` (Optimize tensor operations with einsum)

## Monitoring Commands
```bash
# Check job status
gcloud ai custom-jobs describe projects/371617986509/locations/us-central1/customJobs/825863610487013376 --format=\"value(state)\"

# View logs  
gcloud logging read 'resource.labels.job_id=\"825863610487013376\"' --project learnagentspace --limit 100

# Download videos
gsutil -m cp -r gs://footyai/videos/v2_torch_soccer_20250724_201630/ videos/
```

## Expected Outcome
- Complete 10,000 episode training run
- Generate periodic video recordings showing agent improvement
- Save final model checkpoint to GCS

## Current Status
- ✅ Job deployed successfully
- ✅ Training running on CPU (GPU config issue noted in #8)
- ✅ Videos being generated and uploaded to GCS
- ⏳ Progress: Episode 1,201/10,000
- ⏳ ETA: ~24-48 hours for completion

## Notes
- Running on CPU instead of intended T4 GPU (see issue #8)
- Videos show typical early-stage random behavior
- Training metrics show low but positive rewards
